index.js

```diff
@@ -241,25 +269,22 @@ const nextMatch = async matches => {
   }
 
   if (matches > 1) {
-    await nextMatch(--matches);
+    await nextMatch(matches - 1);
   }
-}
-
-
+};
 
 client.once('ready', () => {
   console.log('Ready!');
 });
 
-
 client.on('ready', async () => {
-
   if (once === true) {
     await nextMatch();
     client.destroy();
   } else {
     // Number of seconds until the next even hour
-    let countdown = ((60 - now.getSeconds()) + 60 * (60 - now.getMinutes()) + 60 * 60 * (1 - now.getHours() % 2));
+    const countdown = ((60 - now.getSeconds()) + 60
```

Comment:
This is a weird line break here. We might want to put some extra parens in or disable linting on this line or something. But fine for now.

Reply:
Leaving as is, then, if you've no alternatives.